todo/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="./style.css">
+   <title>ToDoList</title>
+</head>
+<body>
+   <div class="contaner">
+      <h1>ToDoList</h1>
+         <div class="input">
+            <input type="text" name="todo" id="todo" class="js-input"></input>
+            <button class="js-btn">Add</button>
+         </div>
+      <div class="js-todos"></div>
```

Comment:
лучше сделать через id и потом получить через id в скрипте
плюс лучше это сделать через `ul` список
```suggestion
      <div id="todo" class="js-todos"></div>
```